packages/content-api/lib/index.js

```diff
@@ -35,6 +55,11 @@ export default function GhostContentAPI({host, version, key}) {
     return api;
 
     function makeRequest(resourceType, userParams, id, membersToken = null) {
+        if (!membersToken && !key) {
```

Comment:
This check is weird, because you can create an instance of the content api without a key, but we then require a memberToken for each request